venv/Lib/site-packages/PIL/JpegImagePlugin.py

```
@@ -0,0 +1,902 @@
+    def load_djpeg(self) -> None:
+        # ALTERNATIVE: handle JPEGs via the IJG command line utilities
+
+        f, path = tempfile.mkstemp()
+        os.close(f)
+        if os.path.exists(self.filename):
+            subprocess.check_call(["djpeg", "-outfile", path, self.filename])
+        else:
+            try:
+                os.unlink(path)
+            except OSError:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(OSError)`

(SIM105)